components/ui/card.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react'
+import { cn } from '@/lib/utils'
+import { CardI } from '@/lib/interface/controls/card'
+import Image from 'next/image'
+
+const Card = React.forwardRef<HTMLDivElement, CardI>(
+  (
+    {
+      className,
+      title,
+      children,
+      variant = 'default',
+      primaryColor = 'default',
+      imgUrl,
+      members,
+      subject,
+      desc,
+      Icon,
+      ...props
+    },
+    ref
+  ) => (
+    <>
+      {variant === 'default' && (
+        <div
+          ref={ref}
+          className={cn(
+            'rounded-lg border border-slate-200 bg-white  min-w-[280px] max-w-[300px] h-[160px] text-slate-950 shadow-sm dark:border-slate-800 dark:bg-slate-950 dark:text-slate-50',
+            className
+          )}
+          {...props}
+        >
+          <CardHeader
+            className={`${primaryColor == 'default' ? 'bg-[#3A86FF]' : 'bg-[#EECB51]'} font-medium rounded-t-lg flex-row   flex items-center gap-2`}
+          >
+            <div
+              className={`${primaryColor == 'default' ? 'bg-[#66A1FF]' : 'bg-[#FCEBAF]'}
```

Comment:
Same as the above comment; also, there is a syntax error here. The line below needs to be joined to the line above.